### ISSUE TITLE
webpack-demo app: Bug with search with featureLayer source : "view:no-layerview-for-layer"

### ISSUE THREAD

**Issue by Biboba** [Esri/jsapi-resources#162]
I am reporting an issue with 

- [ ] TypeScript definitions
- [X] another resource in this repository

With the demo webpack application, when using search with sources coming from FeatureLayer, search operation (once result is selected) fails to draw the graphic result and returns an error "view:no-layerview-for-layer".

Steps:
1. Clone and install the demo webpack plugin : 
https://github.com/Esri/jsapi-resources/tree/master/4.x/webpack/demo
2. Define sources in the search:
```js
const search = new Search({ 
      view: this.view,
      sources: [{
        featureLayer: {
          url: "https://services.arcgis.com/V6ZHFr6zdgNZuVG0/arcgis/rest/services/CongressionalDistricts/FeatureServer/0",
        },
        searchFields: ["DISTRICTID"],
        displayField: "DISTRICTID",
        exactMatch: false,
        outFields: ["DISTRICTID", "NAME", "PARTY"],
        name: "Congressional Districts",
        placeholder: "example: 3708",
      }]
    });
```
3. Search for a congessional disctrict and select result
4. Map zooms to result but no graphic is displayed and an error "view:no-layerview-for-layer" is logged in the console

Duplicate: https://github.com/Esri/arcgis-webpack-plugin/issues/34

**Comment by Biboba** [comment]
Actually, I think it is not related to the webpack-build and was able to reproduce it with ESRI built API:
https://codepen.io/anon/pen/KbdzeN

In the codepen below, if you update the "map" property of the mapView from "mapBug" to "mapOK", then graphic result is displayed. 
What is the difference between the 2 maps ? Just the basemap : the working configuration is with an official ESRI basemap ("dark-grey"), the other one is from a portal item.
No error is logged in the console but it is the same phenomena as with the webpack config described in my previous post: no graphic is displayed. I just think it fails silently with the built API.

**Comment by odoe** [comment]
You can work around this by providing a `resultSymbol` to the FeatureLayer source. This happens because the app is using a custom basemap. The other errors you saw about the layerview can be ignored.

**Comment by Biboba** [comment]
Indeed ! Thanks @odoe !
I got confused as the error stack goes through _getHighlightedGraphic, I thought the lack of graphic was due to that ! 
Nevertheless, the error is still there with the webpack build but it does not seem to break anything.

**Comment by odoe** [comment]
This will be fixed in 4.11
